src/js/components/PopoverComponent.js

```diff
@@ -0,0 +1,81 @@
+var React = require("react/addons");
+var classNames = require("classnames");
+
+var Util = require("../helpers/Util");
+
+const DEFAULT_ALIGNED = "default";
```

Comment:
I already thought about using it for the labels... As long as we don't need a right/left aligned popover, we only need to adjust the corresponding css/less. But you're right, adding them as props would give as even more freedom.